ncm-download/src/main/perl/download.pm

```diff
@@ -121,6 +121,9 @@ sub Configure {
             $min_age = $inf->{files}->{$f}->{min_age};
         }
 
+	my $timeout = $inf->{files}->{$f}->{timeout};
+	$timeout = $inf->{timeout} unless defined($timeout) && $timeout == 0;
```

Comment:
why the ` && $timeout == 0`? the `unless defined($timeout)` should be enough (and it should be possible to force `timeout=0` for a file; unless that makes no sense)